### PR TITLE
Mini-rubric updates

### PR DESCRIPTION
**Pull request by maureensturgeon**
- Updates name of tab in TopInstructions to Rubric if there's a rubric
- Updates name of header of rubric to Rubric
- Refactors rubric props for clarity

Screen Shot 2021-06-16 at 2 37 57 PM

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

File: apps/src/templates/instructions/teacherFeedback/TeacherFeedback.jsx [hunk 1]
```diff
@@ -255,6 +247,9 @@ export class TeacherFeedback extends Component {
 
     const displayComment = !!comment || viewAs === ViewType.Teacher;
 
+    const displayCommentSection =
```
Comment (jamescodeorg) [inline apps/src/templates/instructions/teacherFeedback/TeacherFeedback.jsx:250]: Maybe put this above displayComment?  A comment describing why/how these two values are derived might be helpful too.

File: apps/src/templates/instructions/teacherFeedback/Rubric.jsx [hunk 2]
```diff
@@ -28,13 +27,16 @@ class TeacherFeedbackRubric extends Component {
...
     } = this.props;
 
-    const showFeedbackInputAreas =
-      !isReadonly && (!!performance || viewAs === ViewType.Teacher);
+    const studentViewingPerfFeedback =
+      !!performance && viewAs === ViewType.Student;
+
+    const showFeedbackInputAreas = !isReadonly || studentViewingPerfFeedback;
+
+    const expandByDefault = isReadonly && !studentViewingPerfFeedback;
```
Comment (jamescodeorg) [inline apps/src/templates/instructions/teacherFeedback/Rubric.jsx]: This seems confusing.  Is this all to calculate whether we want to expand by default?  If so, could you describe when we want to expand by default and why?

**Review by jamescodeorg — COMMENTED** [review]
Generally looks great, curious about the expand by default logic.

File: apps/src/templates/instructions/teacherFeedback/Rubric.jsx (hunk 2 above)
Comment (maureensturgeon) [reply apps/src/templates/instructions/teacherFeedback/Rubric.jsx]: It is confusing, I was hoping to simplify it. There are several different cases to consider, whether viewing as a teacher or student, whether there is feedback already or if the teacher is giving feedback. I've updated the logic, let me know if it's a bit more clear now.

File: apps/src/templates/instructions/teacherFeedback/Rubric.jsx [hunk 3]
```diff
@@ -27,14 +26,19 @@ class TeacherFeedbackRubric extends Component {
...
-      isReadonly,
-      disabledMode,
+      isEditable,
       onRubricChange,
       viewAs
     } = this.props;
 
-    const showFeedbackInputAreas =
-      !isReadonly && (!!performance || viewAs === ViewType.Teacher);
+    let showFeedbackInputAreas, expandByDefault;
```
Comment (jamescodeorg) [inline apps/src/templates/instructions/teacherFeedback/Rubric.jsx]: Would this be clearer if renamed `expandByDefault` to `expandAllByDefault` or `expandAllRubricFields`?  And maybe add some comments that explain the various cases?  (I think part of the problem is also that I find `performance` to be a confusing name but we can't really change it.)

File: apps/test/unit/templates/instructions/teacherFeedback/TeacherFeedbackTest.jsx [hunk 4]
```diff
@@ -293,8 +287,8 @@ describe('TeacherFeedback', () => {
       expect(rubric.props().rubric).to.equal(
         TEACHER_NO_FEEDBACK_RUBRIC_PROPS.rubric
       );
-      expect(rubric.props().isReadonly).to.equal(true);
-      expect(rubric.props().disabledMode).to.equal(true);
+      expect(rubric.props().isEditable).to.equal(false);
```
Comment (jamescodeorg) [inline apps/test/unit/templates/instructions/teacherFeedback/TeacherFeedbackTest.jsx:289]: These two lines are the same?

File: apps/test/unit/templates/instructions/teacherFeedback/TeacherFeedbackTest.jsx [hunk 5]
```diff
@@ -336,8 +330,8 @@ describe('TeacherFeedback', () => {
         expect(rubric.props().rubric).to.equal(
           STUDENT_NO_FEEDBACK_RUBRIC_PROPS.rubric
         );
-        expect(rubric.props().isReadonly).to.equal(true);
-        expect(rubric.props().disabledMode).to.equal(true);
+        expect(rubric.props().isEditable).to.equal(false);
```
Comment (jamescodeorg) [inline apps/test/unit/templates/instructions/teacherFeedback/TeacherFeedbackTest.jsx:331]: These two lines are the same?

File: apps/test/unit/templates/instructions/teacherFeedback/TeacherFeedbackTest.jsx [hunk 6]
```diff
@@ -138,8 +132,8 @@ describe('TeacherFeedback', () => {
         expect(rubric.props().rubric).to.equal(
           TEACHER_FEEDBACK_RUBRIC_PROPS.rubric
         );
-        expect(rubric.props().isReadonly).to.equal(false);
-        expect(rubric.props().disabledMode).to.equal(false);
+        expect(rubric.props().isEditable).to.equal(true);
```
Comment (jamescodeorg) [inline apps/test/unit/templates/instructions/teacherFeedback/TeacherFeedbackTest.jsx:135]: These two lines are the same?

**Review by jamescodeorg — APPROVED** [review]
Minor comment about the tests, feel free to fix in a future change if you prefer.

File: apps/test/unit/templates/instructions/teacherFeedback/TeacherFeedbackTest.jsx (hunk 4 above)
Comment (maureensturgeon) [reply apps/test/unit/templates/instructions/teacherFeedback/TeacherFeedbackTest.jsx:289]: Great catch!! Thanks, will fix now

**Pull request merged**